specs/icmp.rflx

```diff
@@ -1,15 +1,54 @@
 package ICMP is
 
-   type Tag is (Echo_Reply => 0, Echo_Request => 8) with Size => 8;
-   type Code is range 0 .. 0 with Size => 8;
+   type Tag is (
+      Echo_Reply => 0,
+      Echo_Request => 8,
+      Dest_Unreacheable => 3,
+      Time_Exceeded => 11,
+      Parameter_Problem => 12,
+      Source_Quench => 4,
+      Redirect => 5,
+      Timestamp_Msg => 13,
+      Timestamp_Reply => 14,
```

Comment:
Multiple literals have the same name as message types below. Such conflicts are currently not checked, but lead to errors in the generated SPARK code (#242).

Please add a test to `tests/test_integeration.py` to verify this specification:

```Python
    def test_icmp(self) -> None:
        self.assert_compilable_code([f"{self.specdir}/icmp.rflx"])
```

Reply:
added the test and renamed the messages in b010f98